src/controllers/assetController.ts

```diff
@@ -19,33 +19,46 @@ async function assetController(
   // Request DDO from aquarius
   const ddo = await getDDO(did)
   // Immediately send true response if request is from asset owner
-  ddo.publicKey[0].owner === credentials.value && res.send(true)
-  const ddoCredentials: Credentials = ddo.credentials
-  let userProfile: any
-  if (ddoCredentials === undefined) {
-    // Profile is default allowed if no allow or deny list exists.
-    profileAllowed = true
+  if (!ddo) {
+    console.error('Cannot retrieve DDO')
+    res.send(false)
+    return
   } else {
```

Comment:
since u have return on line 25, then maybe no need else case.
once reached line 26 or 27 automatic has ddo, so can remove 1 layer of braces

same case on line 49 too

just some suggestion to make it look prettier

Reply:
Good suggestion, thanks for that. I've removed the `else` clauses